### DISCUSSION TITLE
is there a way to disable block

### DISCUSSION THREAD

**Discussion by minoseah629** [postmanlabs/postman-flows#435] [Ideas]
hello, 
Is there way to keep a block in the flow, but have the block disabled.  

postman flow is similar to rapidminer in how blocks are connected. but i am not on my machine where its installed to show it.

<img width="294" alt="image" src="https://user-images.githubusercontent.com/1980068/213775616-61488933-8dee-4ba1-9c1c-4b9693d3afd6.png">

**Comment by minoseah629** [comment]
i guess deleting the path to the block does that.  but would still like to have the ability to disable block.

**Reply by samay-v** [reply]
You're right, there isn't a way to 'pause' a path today. Deleting is the only way. Would love to see how RapidMiner is solving this.

**Reply by minoseah629** [reply]
just for posterity, i got a quick video of what looks like in Rapidminer when disabling a block. 
https://user-images.githubusercontent.com/1980068/213966136-61d0be8a-7abd-4069-a853-f24c3015586d.mov
